README.md

```diff
@@ -37,32 +37,28 @@ is beta supported at this time.
 
 | Types        | IonText | IonBinary | Limitations  |
 |:-------------:|:-------------:|:-------------:|:-------------:|
-| null      | yes | no      | none |
-| bool      | yes      | no      |   none |
-| int | yes      | no      |    underscores, binaryints, bigints |
-| float | yes      | no      |    underscores |
-| decimal | yes      | no      |    large fractions are slow roundtrip |
-| timestamp | yes      | no      |    fractional seconds are slow on large fractions |
-| string | yes      | no      |    none |
-| symbol | yes      | no      |    sid0, no symboltokens |
-| blob | no      | no      |    broken |
-| clob | yes      | no      |    backed by string |
-| struct | yes      | no      |    none |
-| list | yes      | no      |    none |
-| sexp | yes      | no      |    none |
-| annotations | yes      | no      |    none |
-| local symbol tables | yes      | no      |    none |
-| shared symbol tables | yes      | no      |    user marshals symboltables into the catalog themselves |
+| null      | yes | yes      | none |
+| bool      | yes      | yes      |   none |
+| int | yes      | yes      |    underscores, binaryints, over a signed 31 bit number >8798240505855 |
```

Comment:
"signed 31-bit number" sounds like 30 bits, plus a sign.  we're supporting 32-bit signed ints, right?

where did the number 8798240505855 come from?  looks too large...

Reply:
yea my b shoulda been 4294967295